prime-dotnet-webapi/prime.csproj

```diff
@@ -1,9 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <RootNamespace>Prime</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <UserSecretsId>2144bc8e-373b-4888-a0ca-b0ff7798bd81</UserSecretsId>
```

Comment:
@itchison This is not a secret, btw. User secrets have an Id because you could have multiple apps with secrets on your machine. This Id could be anything but by convention it's a GUID.